app/src/main/java/edu/card/clarity/presentation/recordReceiptScreen/RecordReceiptScreen.kt

```diff
@@ -59,113 +65,130 @@ fun RecordReceiptScreen(
     val cameraPermissionState = rememberPermissionState(permission = Manifest.permission.CAMERA)
 
     CardClarityTheme {
-        Column(
+        LazyColumn(
```

Comment:
`LazyColumn` is for presenting a huge list of data so that only viewable data are fetched and rendered. Here we only have a fixed number of widgets, so using `Column` is fine.